src/actions/instagram/chats.ts

```
@@ -0,0 +1,215 @@
+    }
+
+    const { data } = await response.json();
+    console.log("data", data[0].participants);
+    
+    if (!data || data.length === 0) {
+      return [];
+    }
+
+    const chats = data.map((conversation: InstagramConversation) => {
+      let lastMessage = "No messages yet";
+      if (conversation.messages && conversation.messages.data && conversation.messages.data.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)